rhel/SPECS/victoriametrics.spec

```diff
@@ -0,0 +1,59 @@
+%define debug_package %{nil}
+
+%define copying() \
+%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
+%license %{*} \
+%else \
+%doc %{*} \
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         VictoriaMetrics
+%global repo            VictoriaMetrics
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          2791008e1982f30892fe86cc98888f4c4519c858
```

Comment:
Maybe we should use release git tags instead of commit hash?

Reply:
This has been copied from [prometheus.spec](https://github.com/percona/pmm-server-packaging/blob/PMM-2.0/rhel/SPECS/prometheus.spec). I'm unsure whether the tag will work properly with `Source0` and `%setup` command.